Sources/ForageSDK/Foundation/Network/LiveForageService.swift

```diff
@@ -231,7 +232,12 @@ extension LiveForageService: Polling {
                         completion(.success(data))
                     /// check message is failed to return error immediately
                     } else if data.failed == true {
-                        completion(.failure(NSError(domain: "Message failed", code: 001, userInfo: nil)))
+                        /// Parse the error returned from SQS message and return it back
+                        let error = data.errors[0]
+                        let statusCode = error.statusCode
+                        let forageErrorCode = error.forageCode
+                        let message = error.message
+                        completion(.failure(ForageError(errors:[ForageErrorObj(httpStatusCode:statusCode, code:forageErrorCode, message:message)])))
```

Comment:
Should line 252 in this file also use ForageError instead of NSError? Or are you punting that to the future?

Reply:
Changed here: [45a5be4](https://github.com/teamforage/forage-ios-sdk/pull/32/commits/45a5be410a006634e95850fdb6b0e36d0dc3c79b)